src/PanZoom.js

```diff
@@ -1,27 +1,23 @@
-// @flow-disabled
+// @flow
```

Comment:
I don't know flow-js at all. I made the changes in this file by looking at the diff between our version and upstream...